File: aula3/desafio.cpp

// usando a estrutura if else-if
#include <iostream>
using namespace std;

int main(){
    string produto;
    cout << "Digite a quantidade em produto:";
    cin >> produto;
    
    int quantidade_estoque;
    cout << "Digite a quantidade do estoque:";
    cin >> quantidade_estoque;

    float preco_unitario;
    cout << "Digite a quantidade do estoque:";
    cin >> preco_unitario;




    if (quantidade_estoque < 1){
        cout << "Status: Em falta.!" << endl;

    } else if (quantidade_estoque < 5 ){
        cout << "Status: Alerta de estoque baixo" << endl;
    }
    else{
        cout << "Status: Estoque OK"  << endl;
    }
    return 0;
}

File: Aula3/desafio.java

// usando a estrutura if else else-if
import java.util.Scanner;

public class Main {
    public static void main(string[]args){
        System.out.print("Digite a quantidade em estouqeu:");
        int quantidade = scanner.nextInt();


        //Usando a estrutura 'if'
        if (quantidade < 1){
            System.out.println("Status: Em falta");
        }else if (quantidade < 5){
            System.out.println("Status: Alerta de estoque baixo");
        }
        else{
            System.out.println("Status: Estoque OK")
        }
        scanner.close();
        }    }
}
